checks.d/kong.py

```diff
@@ -0,0 +1,70 @@
+# stdlib
+import urlparse
+
+# 3rd party
+import requests
+import simplejson as json
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class Kong(AgentCheck):
+    """ collects metrics for Kong """
+    def check(self, instance):
+        metrics = self._fetch_data(instance)
+        for row in metrics:
+            try:
+                name, value, tags = row
+                self.gauge(name, value, tags)
+            except Exception:
+                self.log.error(u'Could not submit metric: %s' % repr(row))
+
+    def _fetch_data(self, instance):
+        if 'kong_status_url' not in instance:
+            raise Exception('missing "kong_status_url" value')
+        tags = instance.get('tags', [])
+        url = instance.get('kong_status_url')
+
+        parsed_url = urlparse.urlparse(url)
+        host = parsed_url.hostname
+        port = parsed_url.port or 8001
+        service_check_name = 'kong.can_connect'
+        service_check_tags = ['host:%s' % host, 'port:%s' % port]
```

Comment:
`host` is actually a reserved keyword : this tag is added by Datadog to specify the host (server running the agent) which sends the service checks. Could you use `kong_host` or something similar instead ?